src/pages/Controllers/index.js

```diff
@@ -232,6 +284,12 @@ class Controllers extends Component {
                   />
                   <AntdDatePicker onChangeCallback={this.fetchControllers} />
                 </Form>
+                <LoginModal
+                  showLoginModal={showLoginModal}
+                  redirect={this.redirectToLoginPage}
+                  content="Please login on to pbench dashboard to bookmark records."
```

Comment:
Can we consider simplifying this message to `Please login to favorite controllers`?

Reply:
yes